modules/custom/moj_resources/src/CategoryFeaturedContentApiClass.php

```diff
@@ -19,7 +19,7 @@ class CategoryFeaturedContentApiClass
    *
    * @var array
    */
-  protected $nids = array();
+  protected $nodeIds = array();
```

Comment:
This is not actually used. A variable with the same name is used locally in one of the functions but not this private one.